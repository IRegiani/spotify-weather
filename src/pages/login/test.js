import React from 'react';
import { shallow } from 'enzyme';
// import ErrorOutlineOutlined from '@material-ui/icons/ErrorOutlineOutlined';

// import { requestLoginAction } from '../../state/actions';

import LoginPage from './index';

describe('LoginPage', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
