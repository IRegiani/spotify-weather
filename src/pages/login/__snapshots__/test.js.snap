// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage renders correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  className="makeStyles-root-1"
  component="main"
  container={true}
>
  <WithStyles(ForwardRef(Button))
    color="primary"
    id="forgotPasswordButton"
    onClick={[Function]}
  >
    Login With Spotify
  </WithStyles(ForwardRef(Button))>
</WithStyles(ForwardRef(Grid))>
`;
